src/index.html

```diff
@@ -12,113 +12,128 @@
-                                    Average</span><span id="caloriestitle-annotation2">Today</span>
-                        </div>
-                        <div>
-                            <p id="subtitle" style="cursor: default;"><span id="caloriessubtitle-annotation1">902 kcal</span><span
-                                    id="caloriessubtitle-annotation2">1437 kcal</span></p>
+                    <div id="line-chart-skeleton"></div>
+                    <div class="chart-padding" id="line" style="display: none;">
+                        <div class="chart-header">
+                            <div id="chart-header-title"><p id="chart-title">Calories Consumed</p></div>
+                            <div class="chart-header-values">
+                                <div id="total-value" style="margin: 0px 10px;">
+                                    <span id="title-annotation"> Total </span>
+                                    <span id="value-annotation" style="color: #780508;"> 1437 Kcal </span>
+                                </div>
+                                <div id="average-value" style="margin: 0px 10px;">
+                                    <span id="title-annotation"> Daily Average </span>
+                                    <span id="value-annotation" style="color: #DB4247;"> 902 Kcal </span>
+                                </div>
+                            </div>
                         </div>
-                        <div id="balance" class="line-chart" style="width: 92%; height:400px;  background: transparent; max-width: 1083px;"></div>
+                        <div id="balance" class="line-chart" style="width: 100%; height:400px;  background: transparent;"></div>
                     </div>
                 </div>
             </div>
             <div class="row">
                 <div class="column-chart-area">
                     <div class="col-sm-12 col-md-4" id="pie-chart" style=" padding: 0px; height: 525px; ">
+                        <div id="pie-chart-skeleton"></div>
+                        <div id="pie-wrapper" style="display: none;">
                         <div class="chart-padding" id="pie">
                             <div id="donut" style="display: block; height: 530px;">
                                 <div class="col-xs-9 col-sm-9 col-md-9 col-lg-9" id="pie" style=" padding: 0px;">
-                                    <div id="pie-title" style=" padding-top: 20px; margin-bottom: 6px; cursor: default; margin-left: 20px; font-family: Roboto;  font-weight: 500; font-size: 20px; text-align: left; color: #1A1A1A">Sunday
-                                        Activity</div>
+                                    <div id="pie-title" style=" padding-top: 20px; margin-bottom: 6px; cursor: default; margin-left: 20px; font-family: Roboto;  font-weight: 500; font-size: 20px; text-align: left; color: #1A1A1A">Macro
```

Comment:
Why you have changed Sunday activity to macro nutrients?

Reply:
Changed Pie-title to correct format.